llabake/Hello-Books
# Ft build admin component 154179187

What does this PR do?
This PR builds the admin dashboard

Description of Task to be completed?
Admin should be able to add a book, accept book borrow and return request and also modify book detail

How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
* on successful login, on navigation to the admin page, the admin dash board should be loaded there
What are the relevant pivotal tracker stories?
154179187

Any background context you want to add?
N/A

Screenshots
![image](https://user-images.githubusercontent.com/29184825/35069279-02db1aa2-fbda-11e7-9ff8-9f85449a2b7c.png)
